gateway/main.go

```diff
@@ -154,6 +162,7 @@ func main() {
 		scaler := scaling.NewFunctionScaler(scalingConfig, scalingFunctionCache)
 		functionProxy = handlers.MakeScalingHandler(faasHandlers.Proxy, scaler, scalingConfig, config.Namespace)
 	}
+	functionProxy = tracing.Middleware(tracing.ConstantName("FunctionProxy"), functionProxy)
```

Comment:
this applies the tracing middleware to only the function proxy, we chould also choose to add it to other endpoints though